templates/partials/reports_list.html

```diff
@@ -8,7 +8,7 @@
   <p ng-show="error" class="alert alert-danger" role="alert">
     <span ng-show="errorSyntax">
       <span translate>invalid.query</span>
-      <a ui-sref="help.search" target="_blank" translate>Help</a>.
+      <a ui-sref="help.search" target="_blank" rel="noopener noreferrer" translate>Help</a>.
```

Comment:
We don't have a route for `help.search` any more so I think this is broken. I guess just delete this line altogether?